src/PackageManagement.VisualStudio/Strings.resx

```diff
@@ -183,4 +183,10 @@
   <data name="Warning_ErrorDuringProjectClosureWalk" xml:space="preserve">
     <value>Failed to resolve all project references for '{0}'. The package restore result for '{1}' may be incomplete.</value>
   </data>
+  <data name="SolutionIsNotAvailable" xml:space="preserve">
+    <value>Solution is not available. Since, there is a packages.config based project and/or globalPackagesFolder setting is a relative path, NuGet needs a saved solution to determine the solution directory. Please save your solution before managing NuGet packages.</value>
```

Comment:
As discussed offline, removed this string. And, simply, used the old string 'Solution is not saved. Please save your solution before managing NuGet packages.'